juju/juju
# Reload interfaces when creating a container on MAAS

## Description of change
When multiple interfaces are created for a container we need to reload device structure for MAAS so that it's complete.

## QA steps
Create a MAAS machine with interfaces in two spaces, create a container on this machine with those two spaces, check that there are 2 interfaces in the container

## Documentation changes
None

## Bug reference
https://bugs.launchpad.net/juju/+bug/1698443
